Call 3rd Party Web Service/Readme.md

```diff
@@ -1,39 +1,37 @@
 # Call 3rd Party Web Service
-This example uses the WebVI to call the [Earthquake API](https://earthquake.usgs.gov/) from the [US Geological Survey](https://www.usgs.gov/) and display recent earthquakes on a webpage.
+This example demonstrates how to use a WebVI to call the [Earthquake API](https://earthquake.usgs.gov/) from the [US Geological Survey](https://www.usgs.gov/) and display recent earthquakes on a web page.
 
-On the block diagram it uses [HTTP GET](http://zone.ni.com/reference/en-XX/help/371361N-01/lvcomm/http_client_get/) to query for earthquakes in the last 30 days and [Unflatten from JSON](http://zone.ni.com/reference/en-XX/help/371361N-01/glang/unflatten_from_json/) to convert the results to LabVIEW data.
+On the diagram, this WebVI uses [HTTP GET](http://zone.ni.com/reference/en-XX/help/371361N-01/lvcomm/http_client_get/) to query the Earthquake API for earthquakes from the last 30 days. The WebVI also uses [Unflatten from JSON](http://zone.ni.com/reference/en-XX/help/371361N-01/glang/unflatten_from_json/) to convert the results from JSON to LabVIEW data.
 
-On the front panel it displays a summary of the results in a data grid indicator and shows a map of the selected earthquake region in a URL Image indicator.
+On the panel, this WebVI displays a summary of the results in a data grid and a map of the selected earthquake region in a URL Image indicator.
 
 ![alt text](Screenshot.PNG "Screenshot")
 
 # Dependencies
 - LabVIEW NXG 2.0 Beta
 
 # Setup
-- Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repo to your machine.
-- Open `Call 3rd Party Web Service/Call 3rd Party WebService.lvproject`
-- Run the WebVI
-  - Open **Main.gviweb** and click the **Run** button
-- Build Web application
-  - Open WebApp.gcomp
-  - Switch to the **Document** tab
-  - Click Build
+1. Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repository to your machine.
+2. Open `Call 3rd Party Web Service/Call 3rd Party WebService.lvproject`
+3. Open `Main.gviweb` and click the **Run** button.
+4. Build the web application.
+  a. Open `WebApp.gcomp`.
```

Comment:
When I view the "a. ... b." lines rendered in GitHub it all shows up on one line